browser/src/StructuralVariantList/StructuralVariants.tsx

```diff
@@ -177,7 +181,7 @@ const StructuralVariants = ({ context, exportFileName, variants }: StructuralVar
     }
   }, [])
 
-  const [colorKey, setColorKey] = useState('consequence')
+  const [colorKey, setColorKey] = useState('type')
   const trackColor = useCallback(
     // @ts-expect-error TS(7006) FIXME: Parameter 'variant' implicitly has an 'any' type.
     (variant) => {
```

Comment:
@rileyhgrant I defined this as a `StructuralVariantPropType` rather than a `StructuralVariant`. As a comment in this PR implies, I took a look at collapsing the two types into one, which turns out to be more finicky than it might seem. Given that we're under the gun I've decided not to touch that for now. Better two partially redundant types than none at all.